simapp/test_helpers.go

```diff
@@ -420,7 +420,7 @@ func NewPubKeyFromHex(pk string) (res crypto.PubKey) {
 	if err != nil {
 		panic(err)
 	}
-	pkEd := make(ed25519.PubKey, ed25519.PubKeySize)
+	pkEd := make([]byte, ed25519.PubKeySize)
 	copy(pkEd, pkBytes)
```

Comment:
the `copy` is not needed. 

You should also check if `pkByte` has correct length.